crates/ty_ide/src/completion.rs

```diff
@@ -357,6 +358,15 @@ fn add_unimported_completions<'db>(
             documentation: None,
         });
     }
+
+    sort_completions(&mut completions);
```

Comment:
Is it now possible that `dedupe_by` won't deduplicate completions that are available both locally and as import or is it guaranteed that local suggestions and unimported completions never overlap?